### DISCUSSION TITLE
Goguardian Goaway

### DISCUSSION THREAD

**Discussion by BreadGhoti** [3kh0/ext-remover#853] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do i use this cna i get a better explanation please thx

**Comment by ghost** [comment]
It's kinda simple but ok
1: Click on the link
1.2: (optional) refresh the page (ctrl+r)
3: Type chrome://restart into your browser, then open chrome again
btw this was literally what the post said

**Reply by BreadGhoti** [reply]
but like what is the link supposed to show?
ss? thx

**Comment by BreadGhoti** [comment]
Please how do i do this and what should i see

**Comment by BreadGhoti** [comment]
allo i get is this 
![Screenshot 2023-12-13 14 27 45](https://github.com/3kh0/ext-remover/assets/148354667/d3913ce8-c394-47a8-a7c0-6f83a6803f91)

**Reply by rwg100** [reply]
skill issue just get better

**Reply by BreadGhoti** [reply]
bro I'm actually asking for help

**Reply by hankypoo7** [reply]
yea bro come on

**Reply by hankypoo7** [reply]
im wanting to know the same thing ill tell you if i find anything

**Comment by BreadGhoti** [comment]
but seriously 3kh0 if you see this a little help please thx

**Reply by Aka-but-nice** [reply]
I found the exploit :skull:?

**Comment by Aka-but-nice** [comment, marked as answer]
And its most likely not working due to tag escaping
